packages/serverless/src/google-cloud-http.ts

```diff
@@ -41,21 +36,10 @@ const _googleCloudHttpIntegration = ((options: { optional?: boolean } = {}) => {
   };
 }) satisfies IntegrationFn;
 
-export const googleCloudHttpIntegration = defineIntegration(_googleCloudHttpIntegration);
-
 /**
  * Google Cloud Platform service requests tracking for RESTful APIs.
- *
- * @deprecated Use `googleCloudHttpIntegration()` instead.
  */
-// eslint-disable-next-line deprecation/deprecation
-export const GoogleCloudHttp = convertIntegrationFnToClass(
-  INTEGRATION_NAME,
-  googleCloudHttpIntegration,
-) as IntegrationClass<Integration>;
-
-// eslint-disable-next-line deprecation/deprecation
-export type GoogleCloudHttp = typeof GoogleCloudHttp;
+export const googleCloudHttpIntegration = defineIntegration(_googleCloudHttpIntegration);
```

Comment:
l: I guess we could inline the integration here now? Same for the others.

Reply:
The main reason to keep this separated (initially) is that I though we may want to export the "unwrapped" version for tests, where we could then use the type safe form - but this is a nice to have, no strong feelings on this!